tensorboard/plugins/profile/profile_plugin.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""The TensorBoard plugin for performance profiling."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import logging
+import os
+import tensorflow as tf
+from werkzeug import wrappers
+
+from tensorboard.backend import http_util
+from tensorboard.backend.event_processing import plugin_asset_util
+from tensorboard.plugins import base_plugin
+from tensorboard.plugins.profile import trace_events_json
+from tensorboard.plugins.profile import trace_events_pb2
+
+# The prefix of routes provided by this plugin.
+PLUGIN_NAME = 'profile'
+
+# HTTP routes
+LOGDIR_ROUTE = '/logdir'
+DATA_ROUTE = '/data'
+TOOLS_ROUTE = '/tags'
```

Comment:
Done. s/tags/tools/

This was "/tags" because the old router provided convenient functions for run+tag combinations.